### PR TITLE
Switch Assets UI

### PR DESCRIPTION
**Pull request by Daniel-Wang**
### Features

- THE APP IS NOW FULLY FUNCTIONAL!!! You can now switch assets and send them!!! 🙌 🎉 

### Concerns

- I'm currently storing `currAssetCode` `currAssetName` and `currAssetIssuer` as variables in WalletApplication.kt, would it be better if they were encapsulated in some "currentAsset" object that lets you access them or is that not needed? (That class probably won't be used again)

- Working through the app I have identified more bugs / refactor opportunities / things I forgot to implement. So I added them to the board.

### Screenshots

screenshot_20181001-175526
screenshot_20181001-180151

### CODE REVIEW

**Comment by ndizazzo** [comment]
This is a monumental milestone! Awesome work @Daniel-Wang 🎉🎉🎉

File: app/src/main/java/blockeq/com/stellarwallet/utils/StringFormat.kt [hunk 1]
```diff
@@ -40,5 +42,18 @@ class StringFormat {
             }
             return false
         }
+
+        fun getAssetCode() : String {
```
Comment (MeghaBambra) [inline app/src/main/java/blockeq/com/stellarwallet/utils/StringFormat.kt]: This seems to be a very app specific function and doesn't need string formatting - we can take it out of the utilities and probably more suitable in a more session object helper (or something similar)

File: app/src/main/java/blockeq/com/stellarwallet/utils/StringFormat.kt [hunk 2]
```diff
@@ -40,5 +42,18 @@ class StringFormat {
...
+            return if (WalletApplication.currAssetCode == Constants.LUMENS_ASSET_TYPE) {
+                Constants.LUMENS_ASSET_CODE
+            } else {
+                WalletApplication.currAssetCode
+            }
+        }
+
+        fun getAvailableBalance(): String {
```
Comment (MeghaBambra) [inline app/src/main/java/blockeq/com/stellarwallet/utils/StringFormat.kt]: Same for this one

**Comment by MeghaBambra** [comment]
@Daniel-Wang getting real close here! Just few comments - otherwise looks great.

File: app/src/main/java/blockeq/com/stellarwallet/WalletApplication.kt [hunk 3]
```diff
@@ -32,11 +34,28 @@ class WalletApplication : MultiDexApplication() {
...
+        var currAssetName: String = Constants.LUMENS_ASSET_NAME
+        var currAssetIssuer: String = ""
+
         var appReturnedFromBackground = false
 
         fun applicationContext(): WalletApplication {
             return instance!!
         }
+
+        fun getAssetCode() : String {
```
Comment (MeghaBambra) [inline app/src/main/java/blockeq/com/stellarwallet/WalletApplication.kt]: @Daniel-Wang Apologies - i should have been more specific. I feel creating another class such as `UserSession` and putting related functions in there would be more suitable. We can also rename `getAssetCode` to `getCurrentAssetCode`. Similarly, `getAvailableBalance` should also be in `UserSession`.

Comment (Daniel-Wang) [reply app/src/main/java/blockeq/com/stellarwallet/WalletApplication.kt]: Okay gotcha!
So that would also have the 3 curr asset vars right?

Comment (MeghaBambra) [reply app/src/main/java/blockeq/com/stellarwallet/WalletApplication.kt]: @Daniel-Wang yes!

Comment (Daniel-Wang) [reply app/src/main/java/blockeq/com/stellarwallet/WalletApplication.kt]: @MeghaBambra how does this look now?

**Pull request merged**